index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio: setup and mobile version skeleton</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav class="toolbar">
+        <h2 class="logo"><a href="#">My Logo</a></h2>
+        <img class="menu-icon" src="images/Union.svg" alt="menu-icon" />
```

Comment:
- [ ] The hamburger icon should be wrapped inside a `<nav>` tag since it will be the one containing the navigation links for the portfolio.